File: output/ChatGPT/codenet/Java/C++/s466963472.cpp
#include <iostream>
using namespace std;

int solve(int d) {
    int ans = 0;
    for (int i = 1; i * d < 600; i++) {
        int x = i * d;
        int y = x * x;
        ans += y * d;
    }
    return ans;
}

int main() {
    int d;
    while (cin >> d) {
        cout << solve(d) << endl;
    }
    return 0;
}

File: output/CodeLlama/codenet/Java/Python/s466963472.py
#!/usr/bin/python
# -*- coding: utf-8 -*-

import sys

def read():
    try:
        d = int(sys.stdin.readline())
        return True
    except:
        return False

def solve():
    ans = 0
    for i in range(1, 600 / d):
        x = i * d
        y = x * x
        ans += y * d
    return ans

while read():
    print solve()